modules/nf-core/mirdeep2/mirdeep2/main.nf

```diff
@@ -0,0 +1,59 @@
+process MIRDEEP2_MIRDEEP2 {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mirdeep2:2.0.1.2--0':
+        'biocontainers/mirdeep2:2.0.1.2--0' }"
+
+    input:
+    tuple val(meta), path(processed_reads), path(genome_mappings)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(mature), path(hairpin)
+
+    output:
+    tuple val(meta), path("result*.{bed,csv,html}")    , emit: outputs
+    path "versions.yml"                                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2.0.1'
+    """
+    miRDeep2.pl \\
+        $processed_reads \\
+        $fasta \\
+        $genome_mappings \\
+        $mature \\
+        none \\
```

Comment:
what is none here? and missing $args

Reply:
Thank you very much for spotting this! The software takes positional arguments and when these are not available they [are replaced by "none"](https://github.com/rajewsky-lab/mirdeep2?tab=readme-ov-file#example-use-2). I have now corrected this and improved the whole module :)